### DISCUSSION TITLE
I waana help translate the steam store page to Hindi.

### DISCUSSION THREAD

**Discussion by NisheshTyagi** [OpenTTD/OpenTTD#12110] [Q&A]
I have access for the hindi translation of the game, but I found out that there's no Hindi steam store page for the game. I want to translate, but I don't have access to do it. I request the devs to create a Hindi language file for the steam store and give me access to modify it.

**Comment by TrueBrain** [comment]
As noted on https://partner.steamgames.com/doc/store/localization/languages, Steam does not offer their Store Pages to be translated in Hindi. As such, we don't offer that, as to not waste anyone's time.

Hope that clear it up :)
